pkg/controller/master/setting/handler.go

```diff
@@ -104,7 +104,11 @@ func (h *Handler) setConfiguredCondition(settingCopy *harvesterv1.Setting, err e
 			return err
 		}
 	} else if err == nil {
-		harvesterv1.SettingConfigured.True(settingCopy)
+		if settingCopy.Value == "" {
+			harvesterv1.SettingConfigured.False(settingCopy)
```

Comment:
@guangbochen, any takes on making the `configured` condition false if the value is empty?

Reply:
looks fine to me since empty value can be treated as not configured.